client/src/pages/AdminPage/ManageSuggestionsPage/SuggestionSearchPage/SuggestionSearchPage.js

```diff
@@ -94,7 +96,7 @@ class SuggestionSearchPage extends React.Component {
 
     async search(changedState) {
         await this.setState({changedState});
-        search(this.state, this.props.history, '/panel/manage/suggestions/search');
+        search({...this.state, ...changedState}, this.props.history, '/panel/manage/suggestions/search');
```

Comment:
What's the point of doing this if there's the `await this.setState({changedState});` line above?

Reply:
Honestly there shouldn't be a difference but this fixed the bug somehow so I'm just going to leave it